src/api/layers/GPXLayer.class.js

```diff
@@ -27,6 +30,7 @@ export default class GPXLayer extends AbstractLayer {
             opacity = 1.0,
             gpxData = null,
             gpxMetadata = null,
+            maxResolution = DEFAULT_MAX_GEOADMIN_RESOLUTION,
```

Comment:
For GPX and KML we should not have any issue if I'm correct ? If yes then we should not have a max resolution for those layers and the AbstractLayer should have a `null` value per default